tests/restart/01-broadcast.t

```diff
@@ -20,18 +20,10 @@ if [[ -z ${TEST_DIR:-} ]]; then
     . $(dirname $0)/test_header
 fi
 #-------------------------------------------------------------------------------
-set_test_number 14
+set_test_number 10
 #-------------------------------------------------------------------------------
 install_suite $TEST_NAME_BASE broadcast
-TEST_SUITE_RUN_OPTIONS=
-SUITE_TIMEOUT=240
-if [[ -n ${CYLC_LL_TEST_TASK_HOST:-} && ${CYLC_LL_TEST_TASK_HOST:-} != 'None' ]]; then
-    ssh $CYLC_LL_TEST_TASK_HOST mkdir -p .cylc/$SUITE_NAME/
-    scp $TEST_DIR/$SUITE_NAME/passphrase $CYLC_LL_TEST_TASK_HOST:.cylc/$SUITE_NAME/passphrase
-    export CYLC_LL_TEST_SITE_DIRECTIVES CYLC_LL_TEST_TASK_HOST
-    TEST_SUITE_RUN_OPTIONS="--set=USE_LOADLEVELER=true"
-    SUITE_TIMEOUT=900
-fi
+cp $TEST_SOURCE_DIR/lib/suite-runtime-restart.rc $TEST_DIR/$SUITE_NAME/
```

Comment:
Rather than copying this file around could we just use a symmlink instead as we do for test_header?